postman-collection/IoT Central.postman_collection.json

```diff
@@ -4188,6 +4424,16 @@
 			"key": "componentId",
 			"value": "componentId-001",
 			"type": "string"
+		},
+		{
+			"key": "dashboardId",
+			"value": "Add your dashboard Id",
```

Comment:
I think we can provide a value here rather than expecting the user to suggest one.